cmdparser.Test/tests.cpp

```diff
@@ -18,14 +18,15 @@ TEST_CASE( "Parse help", "[help]" ) {
 		"--help"
 	};
 
-	Parser parser(2, args);
+	ParserWithPolicy<NoExit> parser(2, args);
 	const auto value = parser.run(output, errors);
 	const std::string prefix = "Available parameters:";
 
 	REQUIRE(parser.has_help() == true);
 	REQUIRE(parser.app_name() == "myapp");
-	REQUIRE(value == false);
-	REQUIRE(output.str().substr(0, prefix.size()) == prefix);
```

Comment:
Sorry cannot accept PRs that break / change existing tests.

Reply:
Isn't that testcase never been reached beforehand anyway as exit() is called?